### PR TITLE
[7.x] Fix for change in chromedriver archive structure

### PR DESCRIPTION
**Pull request by aleahy**
This PR fixes the bug that exists in version 7.x due to the change in the archive structure for the chrome-driver.

### CODE REVIEW

**Review by crynobone — APPROVED** [review]

File: src/Console/ChromeDriverCommand.php [hunk 1]
```diff
@@ -215,20 +215,34 @@ protected function download($version, $os)
      * @param  string  $archive
```
Comment (u01jmg3) [inline src/Console/ChromeDriverCommand.php:214]: ☝️ docblock needs to updated

**Comment by driesvints** [comment]
@aleahy v7 isn't maintained any longer. Please upgrade to v8

**Comment by u01jmg3** [comment]
@driesvints: should GitHub Actions be set to autoclose PRs raised against branches no longer receiving support?

**Comment by driesvints** [comment]
@u01jmg3 I've thought of developing such a flow but haven't found the time yet.

**Comment by driesvints** [comment]
Can maybe become part of https://github.com/laravel/.github/blob/main/.github/workflows/pull-requests.yml as a new `auto-close` flow that closes PR's against branches that aren't the default branch. Also, maintainers should still always be able to send PR's to any branch.

**Pull request merged**